src/pages/Compendia/DownloadCompendia.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { formatSentenceCase, formatBytes } from '../../common/helpers';
+
+import Dropdown from '../../components/Dropdown';
+import Button from '../../components/Button';
+import { useLoader } from '../../components/Loader';
+import Checkbox from '../../components/Checkbox';
+import Spinner from '../../components/Spinner';
+
+import { fetchCompendium, fetchCompendiaData } from '../../api/compendia';
+import { createToken } from '../../state/token';
+import { push } from '../../state/routerActions';
+
+let DownloadCompendia = ({
+  token,
+  createToken,
+  push,
+  title,
+  filter,
+  children,
+}) => {
+  const agreedToTerms = !!token;
+  const [selected, setSelected] = React.useState(null);
+  const [agree, setAgree] = React.useState(agreedToTerms);
+  const { data, isLoading } = useLoader(() => fetchCompendiaData(filter));
+
+  if (isLoading)
```

Comment:
Can we use braces here? [ref](https://github.com/airbnb/javascript#blocks--braces)